protocols/gossipsub/src/generated/gossipsub/pb.rs

```diff
@@ -331,6 +335,38 @@ impl MessageWrite for ControlPrune {
     }
 }
 
+#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
Why don't we want / can derive `Eq`?

Reply:
not sure about this one but it is a generated file using `pb-rs` based on `rpc.proto`

Reply:
Ah ok, I  missed that it's generated.